charts/schulportal-client/templates/deployment.yaml

```diff
@@ -20,9 +21,8 @@ spec:
           imagePullPolicy: {{ default "Always" .Values.image.pullPolicy | quote }}
           ports:
             - name: http
-              containerPort: {{ default 80 .Values.service.port }}
+              containerPort: {{ default 8080 .Values.service.port }}
```

Comment:
when these non-root users attempt to bind to port 80, a well-known HTTP port. This port, like other ports below 1024, is considered privileged and typically requires root access to bind to it. When a Docker container is configured to run as a non-root user and attempts to bind to port 80, it results in a permission denied error.

nginx: [emerg] bind() to 0.0.0.0:80 failed (13: Permission denied)


A solution to this issue involves configuring the container to run on a higher port (above 1024) and then redirecting traffic from port 80 to this higher port